app/src/components/notifications/PageNotifications.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <TmPage
     data-title="My alerts"
-    :loading="$apollo.queries.notifications.loading"
+    :loading="!firstLoaded"
```

Comment:
that's not exactly what i meant - does this even work? it's not using `$apollo.queries.notifications.loading` anymore?

Reply:
just revert this - we will do a refactor later